Components/OrderMailService.php

```diff
@@ -25,8 +25,6 @@ public function __construct(
 
     /**
      * Sends the mail after a payment is confirmed
-     *
-     * @param \Shopware\Models\Order\Order $order
      */
```

Comment:
Any reason to remove the @param completely? Maybe better to adjust the type and var name @bortefi

Reply:
Updated the doc block.

Better would have been to set the type in the method: `sendOrderConfirmationMail(string $orderNumber)`, 
but after a bit of digging I found some inconsistency: 

Methods like `$this->basketService->getOrderByOrderNumber` require an integer for the number,
but the actual data is a string. The Order entity also references the number as a string.
Since no strict typing is applied, this is silently converted, therefore I updated the docblock to string|int.

In the long run this is better replaced by an actual type string and the code adjusted to only pass orderNumber as a string.